doc/payment-api.md

```diff
@@ -89,6 +89,10 @@ Asynchronous calls return an OperationStatus object which is a JSON object with
   * code : number
   * message: error message
 
+Depending on the type of asynchronous call, there may be other key-value pairs.  For example, a z_sendmany operation will also include the following in an OperationStatus object:
```

Comment:
Change Request: Let's require that an `OperationStatus` always includes the `method` field.